src/types.ts

```diff
@@ -101,3 +101,32 @@ export interface Column {
   stylingInfo: string[];
   sortDirection: string;
 }
+
+export interface PageInfo {
+  page: number;
+  rowsPerPage: number;
+  rowsPerPageOptions: number[];
+}
+
+export type SetPageInfo = stardust.SetStateFn<{
+  page: number;
+  rowsPerPage: number;
+  rowsPerPageOptions: number[];
+}>;
+
+export type Announce = ((arg0: AnnounceArgs) => void) | undefined;
+
+export interface RenderProps {
+  direction: 'ltr' | 'rtl';
+  selectionsAPI: stardust.ObjectSelections;
+  rootElement: HTMLElement;
+  tableData: object;
```

Comment:
if you are not doing this in this PR we need to do it at a later point

Reply:
Will get it done when working on handle-data.js

Reply:
Done, may need to do some change working on handle-data.js